src/lib/Syntax.hs

```diff
@@ -75,6 +75,9 @@ data Type = TypeVar TVar
           | NoAnn
             deriving (Show, Eq, Generic)
 
+pattern FixedIntRange :: Int -> Int -> Type
+pattern FixedIntRange low high = IntRange (DepLit low) (DepLit high)
+
```

Comment:
I didn't know about this extension! Very cool. There are lots of places that could use it.